resources/multisite_drupal_standard.make

```diff
@@ -449,7 +454,7 @@ projects[maxlength][subdir] = "contrib"
 projects[maxlength][version] = "3.3"
 
 projects[media][subdir] = contrib
-projects[media][version] = 2.24
+projects[media][version] = 2.26
```

Comment:
I am upgrading this and will reopen the ticket that was closed by the maintainers